src/modules/jobs/job.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Job } from './job.entity';
+
+@Injectable()
+export class JobService {
+  constructor(
+    @InjectRepository(Job)
+    private jobRepository: Repository<Job>
+  ) {}
+
+  async findAll(page: number = 1, pageSize: number = 10): Promise<any> {
```

Comment:
Wrap your points of db in try/catch block for proper error logging.

Reference auth services for guidance